packages/app/src/server/routes/apiv3/users.js

```diff
@@ -899,13 +899,17 @@ module.exports = (crowi) => {
    */
   router.get('/list', accessTokenParser, loginRequired, async(req, res) => {
     const userIds = req.query.userIds || null;
+    const username = req.query.username || null;
```

Comment:
Please add limit option and set any default value.
I've heard that there is some GROWI that has over 1500 active users.

Reply:
Added limit with default value of 20